### ISSUE TITLE
VLC steals focus ONLY on Cinnamon 3.8.8+tara

### ISSUE THREAD

**Issue by flansuse** [linuxmint/Cinnamon#7786]
```
 * Cinnamon version: 3.8.8+tara
* Muffin version: 3.8.2+tara
 * Distribution: Linux Mint 19, 64-bit
 ```

**Issue**
VLC steals focus if it has a video or audio file open (even if paused) if the mouse cursor hovers over any portion of the seekbar. It will steal focus every time, no matter what other applications are in front.

**Steps to reproduce**
Open any file in VLC. Hover the mouse cursor over the seekbar. VLC's interface will be brought above every application and window.

**Expected behaviour**
VLC should not steal focus simply by hovering the mouse cursor.

**Other information**
This bug ONLY exists in Cinnamon 3.8.8+tara with Muffin 3.8.2+tara. It does not occur in any other desktop or previous versions of Cinnamon and Muffin.

Here is the forum post for reference:
https://forums.linuxmint.com/viewtopic.php?f=47&t=273064#p1497784

Here is a video demonstrating the 100% reproducible bug. Notice the mouse cursor and the seekbar. This is not a bug with VLC. This is very likely a bug with the window manager under Cinnamon 3.8.8+tara.

https://trac.videolan.org/vlc/raw-attachment/ticket/20825/VLC3_Qt_Steas_Focus_Mint19.mp4

**Comment by leigh123linux** [comment]
File the issue against vlc as it's base has changed from gtk to qt.

**Comment by mtwebster** [comment]
Caused by https://github.com/linuxmint/muffin/commit/89a586029b03203727e3c95859567763a3455ed0

Interestingly, the tooltip (which is what's causing this) has `_NET_WM_STATE(ATOM) = _NET_WM_STATE_ABOVE, _NET_WM_STATE_STAYS_ON_TOP` when it appears, which is probably what's triggering it.  This is probably always ignored by other wms because the window is not identified as a transient of the parent vlc window.

The linked change was made to prevent the panel from needlessly showing while vlc is fullscreen, when opening a vlc dialog (like file open, etc..)

**Comment by mtwebster** [comment]
Appears fixed in VLC 3.0.3 - try the Flatpak version (the one from mint's upstream is 3.0.2)

**Comment by leigh123linux** [comment]
Looking at vlc-3.0.3 commits 

https://git.videolan.org/?p=vlc/vlc-3.0.git;a=commit;h=f0efb0d42eccb35046d7cee88176ce0d753184d8
